core/src/main/scala/kafka/server/AddPartitionsToTxnManager.scala

```diff
@@ -42,14 +42,26 @@ object AddPartitionsToTxnManager {
   val VerificationTimeMsMetricName = "VerificationTimeMs"
 }
 
+/**
+ * This is an enum which handles the Partition Response based on the Request Version and the exact operation
+ *    defaultError:       This is the default workflow which maps to cases when the Produce Request Version or the Txn_offset_commit request was lower than the first version supporting the new Error Class
+ *    genericError:       This maps to the case when the clients are updated to handle the AbortableTxnException
```

Comment:
I think it's not `AbortableTxnException`. Probably ought to give it its proper name.